pgctl/cli.py

```diff
@@ -347,7 +346,7 @@ def services(self):
             for alias in self.pgconf['services']
             for service_name in self._expand_aliases(alias)
         ]
-        return uniq(services)
+        return tuple(sorted(set(services)))
```

Comment:
This doesn't preserve ordering like the old one, and it's incidental complexity.